scenes/world_map/frays_end.tscn

```diff
@@ -41,6 +41,9 @@
 [ext_resource type="Resource" uid="uid://byhqan5a7vkgg" path="res://scenes/world_map/components/tutorial_npc.dialogue" id="32_djq26"]
 [ext_resource type="Shader" uid="uid://chgaevc31p1mn" path="res://scenes/game_elements/props/clouds_overlay/components/clouds_shadows.gdshader" id="34_opcsp"]
 [ext_resource type="PackedScene" uid="uid://d0c4l7ev6ca3c" path="res://scenes/game_elements/props/spawn_point/spawn_point.tscn" id="37_thm8h"]
+[ext_resource type="Texture2D" uid="uid://b4gjylrs0g2xx" path="res://scenes/game_elements/props/decoration/crochenthemum/ND_Crochenthemum_01.png" id="40_06x6x"]
+[ext_resource type="Texture2D" uid="uid://c8q2r1dsn1m7w" path="res://scenes/game_elements/props/decoration/crochenthemum/ND_Crochenthemum_02.png" id="41_xt3dw"]
+[ext_resource type="Texture2D" uid="uid://72tsg0sif7sq" path="res://scenes/game_elements/props/decoration/mushroom/ND_Mushroom_Blue_01.png" id="42_lftgk"]
```

Comment:
You have not committed these three files to the repo. I believe that if you run `git status` you will see a mention of "untracked files" and the path `scenes/game_elements/props/decoration`. You need to do something like this:

```
git add scenes/game_elements/props/decoration
git commit -m 'Add mushroom and flower assets'
git push
```